library.properties

```diff
@@ -1,10 +1,10 @@
 name=TinyLoRa
-version=1.2.2
-version=1.3.0
+version=1.4.0
 author=Adafruit
 maintainer=Adafruit<info@adafruit.com>
 sentence=Tiny LoRa Library for TTN
 paragraph=Small LoRa library for use with The Things Network
 category=Communication
 url=https://github.com/adafruit/TinyLoRa/
 architectures=*
+depends=DHT sensor library
```

Comment:
@dherrada This library doesn't have a dep. on the DHT sensor lib.?

Reply:
I think that the library doesn't, but some of the examples do. Is there a way to add it in a different spot so it shows up as a dependency for the examples but not the library itself?

Reply:
I don't think so. Just looked at Adafruit IO Arduino which has a lot of examples using libraries. Adds them to `depends` same as here (https://github.com/adafruit/Adafruit_IO_Arduino/blob/master/library.properties#L10).